ldap/servers/slapd/slap.h

```diff
@@ -1722,6 +1722,7 @@ typedef struct conn
     uint64_t c_connid;               /* id of this connection for stats*/
     PRUint64 c_maxthreadscount;      /* # of times a conn hit max threads */
     PRUint64 c_maxthreadsblocked;    /* # of operations blocked by maxthreads */
+    PRBool c_flagblocked;            /* Flag the next read operation as blocked */
```

Comment:
I would rather add the new flag around the end of the struct (less risk to disrupt things like performance impact because alignment changed))
Another point: these days we tend to use plain bool (and true/false) rather than PRBool